frontend/src/pages/host-dashboard/BackgroundPage.tsx

```diff
@@ -0,0 +1,300 @@
+
+  const step2Questions = [
+    {
+      question: 'Do you smoke?',
+      followupQuestion: 'Do you smoke in your house?',
+    },
+    {
+      question: 'Do you drink alcohol?',
+      followupQuestion: 'Do you have any concerns about your drinking?',
+    },
+    {
+      question: 'Do you use any substances?',
+      followupQuestion: 'Do you have any concerns about your substance use?',
+    },
+  ];
+
+  // Handlers for step 2 answer change
+  const handleStep2AnswerChange = (index: number, value: string) => {
+    const updated = [...step2Answers];
+    updated[index].answer = value;
+    if (value === 'No') updated[index].followup = ''; // Clear follow-up if No
+    setStep2Answers(updated);
+  };
+
+  // Handlers for step 2 followup change
+  const handleStep2FollowupChange = (index: number, value: string) => {
+    const updated = [...step2Answers];
+    updated[index].followup = value;
+    setStep2Answers(updated);
+  };
+
+  // Validation functions
+  const validateStep1 = () => {
+    if (!step1Answer) {
+      setError('Please answer the question.');
+      return false;
+    }
+    if (step1Answer === 'Yes' && !step1Followup.trim()) {
+      setError('Please provide details for the follow-up question.');
+      return false;
+    }
+    setError('');
+    return true;
+  };
+
+  const validateStep2 = () => {
+    for (let i = 0; i < step2Answers.length; i++) {
+      const {answer, followup} = step2Answers[i];
+      if (!answer) {
+        setError(`Please answer question ${i + 1}.`);
+        return false;
+      }
+      if (answer === 'Yes' && !followup) {
+        setError(`Please answer the follow-up question for question ${i + 1}.`);
+        return false;
+      }
+    }
+    setError('');
+    return true;
+  };
+
+  const handleNextClick = () => {
+    if (step === 1) {
+      if (!validateStep1()) return;
+      setStep(2);
+    } else {
+      if (!validateStep2()) return;
+      navigate('/host/overview');
+    }
+  };
+
+  const handlePreviousClick = () => {
+    if (step === 2) setStep(1);
+    else navigate('/host/references');
+  };
+
+  return (
+    <PageContainer>
+      <Header>
+        <Typography
+          variant="body2"
+          sx={{cursor: 'pointer'}}
+          onClick={() => navigate('/host/')}
+        >
+          &lt; Back to Dashboard
+        </Typography>
+      </Header>
+
+      <ContentContainer>
+        <Typography
+          variant="body2"
+          sx={{
+            cursor: 'pointer',
+            color: 'primary.main',
+            mb: 2,
+            fontWeight: 500,
+          }}
+          onClick={() => navigate('/host/overview')}
+        >
+          &lt; My Profile Overview
+        </Typography>
+
+        <Typography variant="h4" gutterBottom sx={{fontWeight: 'bold'}}>
+          Please fill out the information below.
+        </Typography>
+
+        <Typography variant="body1" paragraph></Typography>
+
+        <Divider sx={{my: 3}} />
+
+        {step === 1 && (
```

Comment:
This could be a good place for an `enum` to leverage TypeScript to the benefit of code readability, TBD pending discussion, not a blocker for this task